packages/react/src/auto/polaris/PolarisDateTimePicker.tsx

```diff
@@ -130,17 +158,46 @@ export const PolarisDateTimePicker = (props: {
           />
         </div>
       </Popover>
-      {props.includeTime && (
-        <TextField
-          id={props.id ? `${props.id}-time` : undefined}
-          label={props.timeLabel ?? "Time"}
-          prefix={<Icon source={ClockIcon} />}
-          autoComplete="off"
-          value={timeString}
-          onChange={onTimeStringChange}
-          {...timeInputFocusProps}
-          error={!parseSuccess && !isTimeFocused && `Invalid time, please enter a value like "11:00 AM"`}
-        />
+      {(props.includeTime ?? (config as any).includeTime) && (
```

Comment:
The time section is fairly large and separate from the date component. Consider moving it to a separate non-exported component